homeassistant/components/frontend/www_static/home-assistant-polymer

```diff
@@ -1 +1 @@
-Subproject commit 75d760fc3e6f37485ae535a5476c8b00ecd2dfec
+Subproject commit db109f5dda043182a7e9647b161851e83be9b91e
```

Comment:
Hi @MartinHjelmare are you able to help review PR 3932 at all? It seems to have become stuck! Thanks